packages/template-retail-react-app/app/pages/reset-password/reset-password-landing.jsx

```diff
@@ -0,0 +1,105 @@
+import PasswordRequirements from '@salesforce/retail-react-app/app/components/forms/password-requirements'
+import useUpdatePasswordFields from '@salesforce/retail-react-app/app/components/forms/useUpdatePasswordFields'
+import usePasswordReset from '@salesforce/retail-react-app/app/hooks/use-password-reset'
+import useNavigation from '@salesforce/retail-react-app/app/hooks/use-navigation'
+import {
+    API_ERROR_MESSAGE,
+    INVALID_TOKEN_ERROR_MESSAGE
+} from '@salesforce/retail-react-app/app/constants'
+
+const ResetPasswordLanding = () => {
+    const form = useForm()
+    const {formatMessage} = useIntl()
+    const {search} = useLocation()
+    const navigate = useNavigation()
+    const queryParams = new URLSearchParams(search)
+    const email = queryParams.get('email')
+    const token = queryParams.get('token')
+    const fields = useUpdatePasswordFields({form})
+    const password = form.watch('password')
+    const {resetPassword} = usePasswordReset()
+
+    const submit = async (values) => {
+        form.clearErrors()
+        try {
+            await resetPassword({email, token, newPassword: values.password})
+            navigate('/login')
+        } catch (error) {
+            const message = /Unauthorized/i.test(error.message)
+                ? formatMessage(INVALID_TOKEN_ERROR_MESSAGE)
+                : formatMessage(API_ERROR_MESSAGE)
+            form.setError('global', {type: 'manual', message})
+        }
+    }
+
+    return (
+        <Stack justify="center" align="center" spacing={6}>
+            <BrandLogo width="60px" height="auto" />
+            <Stack spacing={2}>
+                <Text align="center" fontSize="xl" fontWeight="semibold">
+                    <FormattedMessage
+                        defaultMessage="Reset Password"
+                        id="reset_password_form.title.reset_password"
+                    />
+                </Text>
+            </Stack>
+            <Container variant="form">
+                <form onSubmit={form.handleSubmit(submit)}>
+                    <Stack spacing={6} paddingLeft={4} paddingRight={4}>
+                        {form.formState.errors?.global && (
+                            <Alert data-testid="password-update-error" status="error">
+                                <AlertIcon color="red.500" boxSize={4} />
+                                <Text fontSize="sm" ml={3}>
+                                    {form.formState.errors.global.message}
+                                </Text>
+                            </Alert>
+                        )}
+                        <Stack spacing={3} pb={2}>
+                            <Field {...fields.password} />
+                            <PasswordRequirements value={password} />
+                        </Stack>
+                        <Button
+                            type="submit"
+                            onClick={() => form.clearErrors('global')}
```

Comment:
This is a little confusing to have a onclick on this submit button knowing that there is also a submit handler on the form. Is it possible to clear these global errors in the other handler?